clients/python/girder_client/cli.py

```diff
@@ -94,6 +95,15 @@ def main():
             print('download command only accepts parent-type of folder')
         else:
             g.downloadFolderRecursive(args.parent_id, args.local_folder)
+            g.saveLocalMetadata(args.local_folder)
```

Comment:
I think we should make the saving of local metadata optional and off by default when using the existing download command so that we don't accidentally break anybody's existing workflows.